core/kernel/classes/class.Resource.php

```diff
@@ -576,7 +576,6 @@ public function getUsedLanguages( core_kernel_classes_Property $property)
      */
     public function duplicate($excludedProperties = array())
     {
-        $returnValue = null;
         $returnValue = $this->getImplementation()->duplicate($this, $excludedProperties);
```

Comment:
Please return directly without a temp variable ! :-)

Reply:
kept this in place as we are likely to add an event in between in the future

Reply:
Ok for me then.